k8s/spinnaker/README.md

```diff
@@ -10,97 +10,117 @@ If you would like to learn more about Spinnaker, please visit the
-domain behind the [Identity-Aware Proxy](https://cloud.google.com/iap/).
+### Networking
+
+As a safe default, the Spinnaker instance is not exposed to external traffic. It's accessed
+via port fowarding, which can be set up with with a single command from the management environment.
+Alternatively, the management environment allows you to expose Spinnaker via the [Identity-Aware Proxy](https://cloud.google.com/iap/), using a secure domain.
 
 # Installation
 
-## Quick install with Google Cloud Shell
+## Quick install via the management console
 
-You must use [Google Cloud Shell](https://cloud.google.com/shell/) to perform the installation.
+You must use the management environment, based on [Cloud Shell](https://cloud.google.com/shell/),
+to perform the installation.
 
 [![Provision Spinnaker](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy.git&cloudshell_working_dir=k8s/spinnaker/scripts/install&cloudshell_tutorial=provision-spinnaker.md)
 
 ### Prerequisites
 
 #### Set up command line tools
 
-Cloud Shell has all the required tools pre-installed.
+The management environment has all the required tools pre-installed.
 
 #### Create a Google Kubernetes Engine cluster
 
-The Spinnaker installation process will create a new GKE cluster by default. If
-you would prefer to use an existing cluster, you must ensure that it has:
+By default, the installation process creates a new GKE cluster for your Spinnaker
+instance. To use an existing cluster, you must ensure that it has:
+
 * IP aliases enabled (since we are using a
-[hosted Redis](https://cloud.google.com/memorystore/) instance)
+[Cloud Memorystore for Redis](https://cloud.google.com/memorystore/) instance)
 * `Full Cloud Platform` scope for its nodes (if using the default service account)
 
-#### Install the Solution
+#### Install Spinnaker for Google Cloud Platform
+
+When you click on the "Open in Google Cloud Shell" button above, the management
+environment asks you to copy/paste and execute a number of commands. Click to copy/paste,
+then press Enter to execute.
+
+As part of this process, a `properties` file is generated. You can modify the
+values in the generated `properties` file to fine-tune your configuration
+_before_ launching the actual `setup` script.
+
+When the initial installation process completes, additional commands will be shown
+to allow you to:
 
-When you click on the "Open in Google Cloud Shell" button above, a tutorial
-pane will open with several copy/paste commands. As part of this process, a
-`properties` file will be generated. You can modify the values in the generated
-`properties` file to fine-tune the installation _prior_ to launching the actual
-`setup` script.
+* find your Spinnaker deployment in the GKE Applications view
+* access Spinnaker via port-forwarding
```

Comment:
@dorbin Where you referring to this place? Add something like "(you should use the provided script, not `hal deploy connect`)?